ml-proto/spec/memory.mli

```diff
@@ -4,11 +4,9 @@
 
 type memory
 type t = memory
-type address = int
```

Comment:
The reason I removed was that, before, we half treated these as opaque types and, with this PR, we fully assume they are int64s, so it seemed like an unnecessary abstraction.  But I wasn't considering the documentation value; I can add them back.